docs/google_drive.md

```diff
@@ -1,11 +1,11 @@
 # Google Drive connector
 
 This connector uploads parquet files to a Google Drive folder.
-This documentation page introduces some prerequesties, and the way to setup:
+This documentation page introduces some prerequesites, and the way to setup:
```

Comment:
prerequisites is the correct spelling

Reply:
Yes, thanks, taken into account in PR #90